docking/pose_dock.py

```diff
@@ -66,11 +70,40 @@ def pose_dock(
     if not os.path.exists(outdir):
         os.makedirs(outdir)
 
+    host_mol = Chem.MolFromPDBFile(host_pdbfile, removeHs=False)
+    amber_ff = app.ForceField("amber99sbildn.xml", "tip3p.xml")
+    host_file = PDBFile(host_pdbfile)
+    host_system = amber_ff.createSystem(
+        host_file.topology,
+        nonbondedMethod=app.NoCutoff,
+        constraints=None,
+        rigidWater=False,
+    )
+    host_conf = []
+    for x, y, z in host_file.positions:
+        host_conf.append([to_md_units(x), to_md_units(y), to_md_units(z)])
+    host_conf = np.array(host_conf)
+
+    final_potentials = []
+    host_potentials, host_masses = openmm_deserializer.deserialize_system(host_system, cutoff=1.2)
+    host_nb_bp = None
+    for bp in host_potentials:
+        if isinstance(bp, potentials.Nonbonded):
+            # (ytz): hack to ensure we only have one nonbonded term
+            assert host_nb_bp is None
+            host_nb_bp = bp
+        else:
+            final_potentials.append(bp)
+
+    padding = 0.1
```

Comment:
note to self and @maxentile  we should really fix this later on. This padding was done to address the particles that are too close to the boundary.

Reply:
Agreed!